pkg/target/target.go

```diff
@@ -46,6 +46,20 @@ autoreject_review[rejection] {
 	}
 }
 
+autoreject_review[rejection] {
+	constraint := {{.ConstraintsRoot}}[_][_]
+	spec := get_default(constraint, "spec", {})
+	match := get_default(spec, "match", {})
+	has_field(match, "namespaceSelector")
+	{{.DataRoot}}.cluster["v1"]["Namespace"]
```

Comment:
this line should be dropped (wont trigger if namespace isn't cached)

No need to add a replacement line as the below `not` statement takes care of checking for the existence of `Namespace`